dsc/tests/dsc_include.tests.ps1

```diff
@@ -203,9 +203,39 @@ resources:
 
         $out = dsc config set -d $includeConfig | ConvertFrom-Json
         $LASTEXITCODE | Should -Be 0
-        $out.results[0].result[0].name | Should -Be 'one'
-        $out.results[0].result[0].type | Should -Be 'Test/Echo'
-        $out.results[0].result[0].result.afterState.output | Should -Be 'Hello World'
+        $out.results[0].result.beforeState[0].name | Should -Be 'one'
+        $out.results[0].result.beforeState[0].type | Should -Be 'Test/Echo'
+        $out.results[0].result.afterState[0].result.afterState.output | Should -Be 'Hello World'
         $out.hadErrors | Should -Be $false
     }
+
+    It 'Test with include works' {
+        $includeYaml = Join-Path $PSScriptRoot ../../dsc/examples/include.dsc.yaml
```

Comment:
I wanted to make sure wherever you execute this test script from, it finds the right config file so the path is relative to `$PSScriptRoot` rather than current working directory